src/compas_viewer/config.py

```diff
@@ -238,7 +240,17 @@ class StatusbarConfig(ConfigBase):
 class SidebarConfig(ConfigBase):
     show: bool = True
     sceneform: bool = True
-    items: list[dict[str, str]] = None
+    items: list[dict[str, str]] = field(
+        default_factory=lambda: [
+            {
+                "type": "Sceneform",
+                "action": {
+                    "Name": get_name,
+                    "Show": get_show_box,
```

Comment:
```python
"Name": lambda obj: obj.name,
"Show": {
    "type": "checkbox"
    "getter": lambda obj: obj.show,
    "setter": lambda obj, value: obj.show = value  
}
```

Reply:
```
{
    "type": "Sceneform",
    "action": [
        {"title": "Name", "action": get_obj_attr_cmd, "kwargs": {"attr": "name"}},
        {"title": "Show", "type": "checkbox", "action": get_obj_attr_cmd, "kwargs": {"attr": "show"}},
    ],
}
```
I found another solution that has similar structure than other :)